hw4_Zadacha5.py

```diff
@@ -0,0 +1,29 @@
+# Каждый из N школьников некоторой школы знает Mi языков.
+# Определите, какие языки знают все школьники и языки,
+# которые знает хотя бы один из школьников.
+
+pupils = int(input())
+lang_list1 = []
+all_lang = set()
+
+for i in range(pupils):
+    lang = int(input())
+    lang_list = set()
+    lang_list1.append(lang_list)
+    for u in range(lang):
+        q = input()

```

Comment:
what is the q and u and where we overall use u? 
for _ in ...

Reply:
fixed